dynamic.js

```diff
@@ -5,6 +5,8 @@ const path = require('path')
 const yaml = require('js-yaml')
 
 module.exports = function (fastify, opts, next) {
+  const fastifySchemas = fastify.getSchemas()
```

Comment:
I have some concerns about using `fastify.getSchemas()`, cause they will return schemes only from "current" scope. So in this fix will only support those schemes which are defined through fastify plugin.

In order to support all cases you'll probably need to get them recursively from all contexts. See https://github.com/SkeLLLa/fastify-oas/blob/master/lib/openapi/index.js#L5 for example.